src/Services/DependencyInjection/ContainerFactory.php

```diff
@@ -269,8 +272,29 @@ private static function autoRegisterClass(ContainerBuilder $container, $classNam
         $reflectionClass = new ReflectionClass($className);
         $constructor = $reflectionClass->getConstructor();
         $definition = $container->register($className, $className);
+
+        /*
+        if (!$constructor) {
+            var_dump($className);
+            exit;
+        }
+        */
+
+        if (!$constructor) {
+            // $reflectionClass = $reflectionClass->gettype();
+            if ($reflectionClass) {
+                $definition->addArgument($reflectionClass->getName());
```

Comment:
not sure what is reason for adding `reflectionClass` as arg to `definition` only in case absence of constructor... Can you clarify it a bit please?

Reply:
Yes,
the goal is to `adding reflectionClass as arg to the definition only in case absence of constructor`.
SF6 change libraries and class. So the autoRegisterClass()  class is through error. I am trying to find way to fix it.

I will remove my code in the next PR so you can easily check error.